microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -19,11 +19,11 @@ WebView2 is available for the following frameworks (or platforms) and programmin
    * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
```

Comment:
I would also remove the /C# from .NET since the ref docs should be applicable to all .NET languages not just C#